src/views/results/index.ts

```diff
@@ -33,17 +33,17 @@ class ResultSetPanelProvider {
     webviewView.webview.html = html.getLoadingHTML();
     this._view.webview.onDidReceiveMessage(async (message) => {
       if (message.query) {
-        const instance = await getInstance();
-        const content = instance.getContent();
-        const config = instance.getConfig();
         let data = [];
 
         let queryObject = Query.byId(message.queryId);
         try {
-          if (undefined === queryObject) {
-            let query = await JobManager.getPagingStatement(message.query, { isClCommand: message.isCL });
+          if (queryObject === undefined) {
+            Query.cleanup();
```

Comment:
We will need to revisit this if we ever allow multiple result tabs like ACS does
(for now, a code comment indicating such is a good idea)